frontend/public/features.ts

```diff
@@ -131,30 +122,65 @@ const detectCanCreateProject = dispatch => coFetchJSON(projectRequestPath)
 export const featureActions = [
   detectCalicoFlags,
   detectOpenShift,
-  detectProjectsAvailable,
   detectCanCreateProject,
 ];
 
-// generate additional featureActions
-[
-  [FLAGS.CAN_LIST_NS, { resource: 'namespaces', verb: 'list' }],
-  [FLAGS.CAN_LIST_NODE, { resource: 'nodes', verb: 'list' }],
-  [FLAGS.CAN_LIST_PV, { resource: 'persistentvolumes', verb: 'list' }],
-  [FLAGS.CAN_LIST_STORE, { group: 'storage.k8s.io', resource: 'storageclasses', verb: 'list' }],
-  [FLAGS.CAN_LIST_CRD, { group: 'apiextensions.k8s.io', resource: 'customresourcedefinitions', verb: 'list' }],
-].forEach(_.spread((FLAG, resourceAttributes) => {
+const projectListPath = `${k8sBasePath}/apis/project.openshift.io/v1/projects?limit=1`;
+const detectShowOpenShiftStartGuide = (dispatch, canListNS: boolean = false) => {
+  // Skip the project check if we know the user can list all namespaces. This
+  // avoids potentially listing thousands of projects more than once (projects
+  // dropdown and flag check). Even though we only ask for one project, the
+  // projects API currently doesn't support paging.
+  //
+  // TODO: Consider adding a global watch for projects / namespaces, which
+  // could remove the need for this flag entirely. It would also prevent us
+  // from re-listing projects when switching from a namespace-scoped resource
+  // to a cluster-scoped resource and back.
+  if (canListNS) {
```

Comment:
I think that `Projects`/`Namespaces` are valuable enough resources to be globally available in Redux for all components. I'm working on adding types to our Redux store so it's more obvious what could be available at compile-time.